packages/fast-colors/app/src/PaletteDisplay/ColorSwatch.tsx

```diff
@@ -0,0 +1,78 @@
+/*---------

```

Comment:
We typically do not leave these types of comments (i.e. "IMPORTS" and "JSS STYLES")

If we do have comments above an interface or function, it would be formatted like:
```
/**
 * Comment text
 */
interface foo {}
```